client/landing/stepper/declarative-flow/internals/steps-repository/importer/index.tsx

```diff
@@ -213,11 +232,62 @@ export function withImporterWrapper( Importer: ImporterCompType ) {
 					fromSite={ fromSite }
 					urlData={ fromSiteData ?? undefined }
 					stepNavigator={ stepNavigator }
+					renderHeading={ ! useContainerV2 }
 				/>
 			);
 		};
 
 		const importJob = getImportJob( importer );
+
+		if ( useContainerV2 ) {
+			const importerData = getImportDragConfig( importer, stepNavigator?.supportLinkModal );
+			const showHeading =
+				importJob?.importerState === appStates.MAP_AUTHORS ||
+				importJob?.importerState === appStates.READY_FOR_UPLOAD ||
+				importJob?.importerState === appStates.UPLOAD_PROCESSING ||
+				importJob?.importerState === appStates.UPLOADING;
+
+			const showBackButton = importJob?.importerState !== appStates.IMPORT_SUCCESS;
+			return (
+				<>
+					<QuerySites siteId={ siteId } />
+					<DocumentHead title={ __( 'Import your site content' ) } />
+					<Interval onTick={ fetchImporters } period={ EVERY_FIVE_SECONDS } />
+					<Step.CenteredColumnLayout
+						className={ clsx(
+							'import__onboarding-page',
+							'importer-wrapper',
+							'step-container-v2--site-migration-identify',
```

Comment:
Do we need this site-migraiton-identify class?

Reply:
good catch!